source/validate_type_unique.cpp

```diff
@@ -36,7 +38,12 @@ spv_result_t TypeUniquePass(ValidationState_t& _,
     }
 
     if (!_.RegisterUniqueTypeDeclaration(*inst)) {
-      return _.diag(SPV_ERROR_INVALID_DATA)
+      // TODO(atgoo@github) Error logging temporarily disabled because it's
+      // failing vulkancts tests. Message in the diagnostics is for unit tests.
+      fprintf(stderr, "WARNING: Duplicate non-aggregate type declarations are"
```

Comment:
I filed https://github.com/KhronosGroup/SPIRV-Tools/issues/568 to fix the command line tool.